doc/src/index.md

```diff
@@ -115,14 +115,25 @@ Julia aims to create an unprecedented combination of ease-of-use, power, and eff
 language. In addition to the above, some advantages of Julia over comparable systems include:
 
   * Free and open source ([MIT licensed](https://github.com/JuliaLang/julia/blob/master/LICENSE.md))
+  * Reliable built-in package manager that automatically installs compatible dependencies
+  * Talented, active, and welcoming community of developers
   * User-defined types are as fast and compact as built-ins
   * No need to vectorize code for performance; devectorized code is fast
   * Designed for parallelism and distributed computation
   * Lightweight "green" threading ([coroutines](https://en.wikipedia.org/wiki/Coroutine))
   * Unobtrusive yet powerful type system
+  * Reusability and composability—it is often possible to pass objects created
+    by one package to generic functions defined by a different author
   * Elegant and extensible conversions and promotions for numeric and other types
   * Efficient support for [Unicode](https://en.wikipedia.org/wiki/Unicode), including but not limited
     to [UTF-8](https://en.wikipedia.org/wiki/UTF-8)
   * Call C functions directly (no wrappers or special APIs needed)
   * Powerful shell-like capabilities for managing other processes
   * Lisp-like macros and other metaprogramming facilities
+
+### [Disadvantages of Julia](@id man-disadvantages-of-julia)
+
+  * It is difficult to generate self-contained binaries or libraries such as .exe, .dll, .so,
+    and impossible to generate small self-contained binaries with very fast runtimes
```

Comment:
```suggestion
    and prohibitively difficult to generate small self-contained binaries with very fast runtimes
```

That said, I don't think this particular framing makes a ton of sense - it feels like we're saying we're giving up on these things, when that's very much not the case - they're being worked on and are not fundamental tradeoffs. I think a framing of "here's some limitations of the current implementation that you should be aware of when evaluating julia" is what we're going for here, but "disadvantages" doesn't really convey that for me.

Reply:
Does [Add header text](https://github.com/JuliaLang/julia/pull/52444/commits/3c901e63e4d23c44497caf3252ef38530427ae29) adequately address this? I do like the primary header being "disadvantages" as a counterpoint to "advantages" and I do think these things are actually disadvantages for folks considering using Julia (even if not for PL theorists)